gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -583,7 +677,9 @@ def create_network_function_device(self, event):
                      {'device': device})
```

Comment:
In this sharing case we need to call HEALTH_MONITOR_COMPLETE, with nfp_context as data now.